libs/ui/src/button.tsx

```diff
@@ -294,9 +294,18 @@ function disabledStyles(p: ThemedStyledButtonProps): CSSObject {
     };
   }
 
+  const backgroundColor = isInverse(color)
+    ? colors.foreground
+    : colors.background;
+  const foregroundColor = isInverse(color)
+    ? colors.background
+    : colors.foreground;
```

Comment:
`colors.foreground` is deprecated. Sorry I didn't mark it deprecated - I was planning to just remove it soon. Instead use `colors.onBackground`.

Reply:
Thanks for the heads-up! Will add the deprecation notice while I'm here